trieste/utils/objectives.py

```diff
@@ -32,7 +32,6 @@ def branin(x: TensorType) -> TensorType:
     """
     The Branin-Hoo function, rescaled to have zero mean and unit variance over :math:`[0, 1]^2`. See
     :cite:`Picheny2013` for details.
-
```

Comment:
Apparently no errors. I guess I should really try to understand the docs format at some point rather than just rely on fixing stuff from the warnings!